docs_user/modules/proc_performing-a-fast-forward-upgrade-on-compute-services.adoc

```diff
@@ -95,35 +95,40 @@ spec:
     - nova
 EOF
 ----
++
+[NOTE]
+The service included in the `servicesOverride` key must match the name of the service that you included in the `OpenStackDataPlaneNodeSet` CR. For example, if you use a custom service called `nova-custom`, ensure that you add it to the `servicesOverride` key.
 
-. Wait for Compute data plane service to be ready:
+. Wait for the Compute data plane services to be ready:
 +
 ----
-oc wait --for condition=Ready openstackdataplanedeployment/openstack-nova-compute-ffu --timeout=5m
+$ oc wait --for condition=Ready openstackdataplanedeployment/openstack-nova-compute-ffu --timeout=5m
 ----
 
 . Run Compute database online migrations to complete the fast-forward upgrade:
 +
 ----
-oc exec -it nova-cell0-conductor-0 -- nova-manage db online_data_migrations
-oc exec -it nova-cell1-conductor-0 -- nova-manage db online_data_migrations
+$ oc exec -it nova-cell0-conductor-0 -- nova-manage db online_data_migrations
+$ oc exec -it nova-cell1-conductor-0 -- nova-manage db online_data_migrations
 ----
 
-. Discover Compute hosts in the cell:
+.Verification
+
+. Discover the Compute hosts in the cell:
 +
 ----
-oc rsh nova-cell0-conductor-0 nova-manage cell_v2 discover_hosts --verbose
+$ oc rsh nova-cell0-conductor-0 nova-manage cell_v2 discover_hosts --verbose
 ----
 
-. Verify if Compute services can stop the existing test VM instance:
+. Verify if the Compute services can stop the existing test VM instance:
```

Comment:
what happens if you can't do this?

Reply:
@bogdando Can you help answer this question? ^